e2e/setup/scenarios.screenshot.ts

```diff
@@ -29,14 +24,14 @@ cucumber.defineRule('take a screenshot of the element', async ({elements, screen
   screenshots.push(await element.screenshot());
 });
 
-cucumber.defineRule('check if the screenshot is exactly equal to the snapshotted version', ({screenshots}: TestEnv) => {
+cucumber.defineRule('check if the screenshot is exactly equal to the snapshotted version', async ({screenshots}: TestEnv) => {
   if (!screenshots.length) throw new Error('E2E: there is no any screenshot, make sure you have "Take a screenshot" before');
   const image = screenshots.pop();
-  expect(image).toMatchImageSnapshot(DIFF_CONFIG);
+  await expect(image).toMatchImageSnapshot();
```

Comment:
Is that aligned with jest concept, is that ok that we have async expect check?

Reply:
> Is that aligned with jest concept, is that ok that we have async expect check?

Yes,
The reason for adding await, is because I've started to notice that error message would direct to the test one after the actual failed one